imgui.ini

```diff
@@ -0,0 +1,35 @@
+[Window][Debug##Default]
```

Comment:
Huh, I've never seen the dashboard create this file. Let's remove it from git and add something to gitignore